generated/nidcpower/nidcpower.proto

```diff
@@ -104,7 +94,6 @@ service NiDCPower {
   rpc ConfigureVoltageLevelRange(ConfigureVoltageLevelRangeRequest) returns (ConfigureVoltageLevelRangeResponse);
   rpc ConfigureVoltageLimit(ConfigureVoltageLimitRequest) returns (ConfigureVoltageLimitResponse);
   rpc ConfigureVoltageLimitRange(ConfigureVoltageLimitRangeRequest) returns (ConfigureVoltageLimitRangeResponse);
-  rpc ConnectInternalReference(ConnectInternalReferenceRequest) returns (ConnectInternalReferenceResponse);
```

Comment:
Is this one a cal api?

Reply:
Yes
![image](https://user-images.githubusercontent.com/47028785/122334913-14533080-cf58-11eb-86be-ef781b5b5f78.png)